UnitySDK/Assets/ML-Agents/Editor/Tests/MLAgentsEditModeTest.cs

```diff
@@ -80,7 +80,7 @@ public TestSensor(string n)
             sensorName = n;
         }
 
-        public int[] GetFloatObservationShape()
+        public int[] GetObservationShape()
```

Comment:
Renamed this in ISensor since the shape applies to non-floats too (that's been bugging me).